core/src/main/java/org/elasticsearch/rest/action/admin/indices/RestIndicesStatsAction.java

```diff
@@ -157,7 +158,7 @@ public boolean canTripCircuitBreaker() {
         return false;
     }
 
-    private static final Set<String> RESPONSE_PARAMS = Collections.singleton("level");
+    private static final Set<String> RESPONSE_PARAMS = new HashSet<String>(Arrays.asList("level", "include_segment_file_sizes"));
```

Comment:
This isn't the right fix, it's not a response parameter (it controls the request that the node client sends). Rather, this parameter needs to be consumed (and parsed as a boolean).